### PR TITLE
add recipe for wdl-aid

### PR DESCRIPTION
**Pull request by DavyCats**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rhpvorderman** [comment]
I agree to be a recipe maintainer.

**Comment by DavyCats** [comment]
@conda-forge/staged-recipes Hi, I'm hitting a bit of a wall here. One of the dependencies of this python package seems to depend on a package that is not available for windows (pygtail), as such the windows tests will fail. Adding `skip: true #[win]` would solve this, but then the linter complains. Either that noarch packages shouldn't have selectors or that python packages have to be noarch (if I remove the noarch part).

Any advice on how to proceed?

File: recipes/wdl-aid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: defdfcecb95c77d8ef3b7d04db878f430bd88019d159f710c8d6042361594b08
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - wdl-aid=wdl_aid.wdl_aid:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```
Comment (chrisburr) [inline recipes/wdl-aid/meta.yaml]: It's okay for `noarch: python` packages to fail on Windows due to missing dependencies as they're built on Linux. If the missing package ever adds Windows support it should (🤞) just work.
```suggestion
```

File: recipes/wdl-aid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - jinja2
```
Comment (chrisburr) [inline recipes/wdl-aid/meta.yaml]: You probably only need `python` and `pip` in `host`

**Comment by DavyCats** [comment]
@chrisburr Thanks!

**Pull request merged**